test/linux/kdialog_handler_test.dart

```diff
@@ -23,7 +23,7 @@ void main() {
       expect(
         dialogHandler.fileTypeToFileFilter(FileType.audio, null),
         equals(
-            'Audio File (*.[aA][aA][cC] *.[mM][iI][dD][iI] *.[mM][pP]3 *.[oO][gG][gG] *.[wW][aA][vV])'),
+            'Audio File (*.[aA][aA][cC] *.[mM][iI][dD][iI] *.[mM][pP]3 *.[oO][gG][gG] *.[wW][aA][vV] *.[mM]4[aA])'),
```

Comment:
This test was not updated in https://github.com/miguelpruivo/flutter_file_picker/pull/1587